Apps/W1/HybridGP/app/src/Migration/Support/HelperFunctions.codeunit.al

```diff
@@ -1342,6 +1344,21 @@ codeunit 4037 "Helper Functions"
                 codeunit.Run(codeunit::"Gen. Jnl.-Post Batch", GenJournalLine);
     end;
 
+    local procedure SafePostGLBatch(JournalBatchName: Code[10])
+    var
+        GenJournalLine: Record "Gen. Journal Line";
+    begin
+        GenJournalLine.SetRange("Journal Template Name", GeneralTemplateNameTxt);
+        GenJournalLine.SetRange("Journal Batch Name", JournalBatchName);
+        if GenJournalLine.FindFirst() then begin
```

Comment:
FindSet, there will be multiple lines to post

Reply:
Check if the codeunit does a findset itself

Reply:
The posting Codeunit uses the first line to determine the batch, then it gets all of that batch's lines. That part of the migration code hasn't changed.